OsmAnd/src/net/osmand/plus/FavouritesDbHelper.java

```diff
@@ -682,6 +842,45 @@ protected void createDefaultCategories() {
 		addEmptyCategory(context.getString(R.string.shared_string_others));
 	}
 
+	public FavoriteGroup createPersonalGroup() {
+		FavoriteGroup group = new FavoriteGroup();
+		group.name = PERSONAL;
+		group.visible = true;
+		group.color = 0;
+		FavouritePoint fPoint;
+		TargetPointsHelper helper = context.getTargetPointsHelper();
```

Comment:
TargetPointsHelper should not deal anymore with Home/Work! This code should be deleted